src/components/families/FamilyDetailsSidebar.tsx

```diff
@@ -80,6 +86,18 @@ const FamilyDetailsSidebar = ({
               {family.parent.information[parentField.id]}
             </Typography>
           ))}
+          <Typography variant="body2" className={classes.heading}>
+            Notes
+          </Typography>
```

Comment:
based on the figma this looks like it should be the same as the basic information header - can we make these both `variant=h3` and apply the styling to both?